mapdata/WarcraftLegacies/UnitData/Skin/1329803368.json

```
@@ -43,7 +43,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Frostwolf Animal Tammer.|n|cfff5962dAbilities:|R Charge, War Cadence|n |n|cff99b4d1Can only train 2.|n|cffffcc00Attacks land and air units.|r"
+      "Value": "Frostwolf Animal Tammer.|n|cfff5962dAbilities:|R Crippling Shout, War Cadence|n |n|cff99b4d1Can only train 2.|n|cffffcc00Attacks land and air units.|r"
```

Could we fix the typos while we're at it, apparently I missed them:
```suggestion
      "Value": "Frostwolf animal tamer.|n|cfff5962dAbilities:|R Crippling Shout, War Cadence|n |n|cff99b4d1Can only train 2.|n|cffffcc00Attacks land and air units.|r"
```